api_tests/collections/test_views.py

```diff
@@ -1822,6 +1822,16 @@ def test_post_mixed_nodes(self):
         assert_not_in(self.contributor_node._id, ids)
         assert_in(self.linked_node._id, ids)
 
+    def test_post_node_already_linked(self):
+        number_of_links = len(self.collection.nodes)
+        res = self.app.post_json_api(
+            self.url, self.payload([self.linked_node._id]),
+            auth=self.user.auth
+        )
+
+        assert_equal(res.status_code, 201)
```

Comment:
This is suspect: why is the status code 201 (CREATED) if no nodes actually get linked? I would expect a 200.